assets/config/assets.onj

```diff
@@ -251,8 +255,48 @@ textures: [
         file: "textures/map/decorations/grass.png"
     },
     {
-        name: "map_decoration_cactus",
-        file: "textures/map/decorations/cactus.png"
+        name: "map_decoration_wasteland_skull_1",
+        file: "textures/map/decorations/wasteland_skull_1.png"
+    },
+    {
+        name: "map_decoration_wasteland_skull_2",
+        file: "textures/map/decorations/wasteland_skull_2.png"
+    },
+    {
+        name: "map_decoration_wasteland_dune_1",
+        file: "textures/map/decorations/wasteland_dune_1.png"
+    },
+    {
+        name: "map_decoration_wasteland_dune_2",
+        file: "textures/map/decorations/wasteland_dune_2.png"
+    },
+    {
+        name: "map_decoration_wasteland_dune_3",
+        file: "textures/map/decorations/wasteland_dune_3.png"
+    },
+    {
+        name: "map_decoration_wasteland_dune_4",
+        file: "textures/map/decorations/wasteland_dune_4.png"
+    },
+    {
+        name: "map_decoration_wasteland_cactus_1",
+        file: "textures/map/decorations/wasteland_cactus_1.png"
+    },
+    {
+        name: "map_decoration_wasteland_cactus_2",
+        file: "textures/map/decorations/wasteland_cactus_2.png"
+    },
+    {
+        name: "map_decoration_bewitchedForest_sheep1",
```

Comment:
the sheeps are the only decorations that don't have an underscore before the number